guides/source/configuring.md

```diff
@@ -700,7 +700,7 @@ Defaults to `'signed cookie'`.
 
 * `config.action_view.debug_missing_translation` determines whether to wrap the missing translations key in a `<span>` tag or not. This defaults to `true`.
 
-* `config.action_view.form_with_generates_remote_forms` determines whether `form_with` generates remote forms or not. This defaults to `true`.
+* `config.action_view.form_with_generates_remote_forms` determines whether `form_with` generates remote forms or not. This defaults to `false`.
```

Comment:
I'm in favor of dropping "This defaults to ..." here.  I would like to continue the pattern from #39971, such that version-specific defaults are only mentioned in version-specific sections.  I think it helps prevent confusion when dealing with multiple versions of Rails / Rails defaults.

Reply:
:+1: I agree and removed the phrase.